account_move_ordering_contacts/models/account_move.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    ordering_contact_ids = fields.Many2many(
```

Comment:
the name of the module and the field seems wrong. It should be something like `orginal_partner_ids` or `origin_partner_ids` or.... you got the point ;)

Reply:
@simahawk both module and fields are now renamed

Reply:
module name should be singular ;)